crates/ty_python_semantic/src/types/function.rs

```diff
@@ -623,11 +623,7 @@ impl<'db> FunctionLiteral<'db> {
     /// calling query is not in the same file as this function is defined in, then this will create
     /// a cross-module dependency directly on the full AST which will lead to cache
     /// over-invalidation.
-    fn signature<'a>(
-        self,
-        db: &'db dyn Db,
-        type_mappings: &'a [TypeMapping<'a, 'db>],
-    ) -> CallableSignature<'db>
+    fn signature<'a>(self, db: &'db dyn Db) -> CallableSignature<'db>
     where
         'db: 'a,
     {
```

Comment:
```suggestion
    fn signature(self, db: &'db dyn Db) -> CallableSignature<'db> {
```

(Surprised Clippy didn't complain about this!)